frontend/styles/Navbar.module.css

```diff
@@ -0,0 +1,426 @@
+  display: block;
+}
+
+.dropdownContainer:hover .linkContent {
+  color: var(--red) !important;
+}
+
+.dropdownContainer:hover .linkUnderline {
+  width: 100%;
+  opacity: 1;
+}
+
+.dropdownContainer:hover .dropdownContent {
+  visibility: visible;
+  opacity: 1;
+}
+
+.dropdownContainer:hover .clearDropdownContent {
+  visibility: visible;
+  opacity: 1;
+}
+
+.dropdownContent {
+  position: absolute;
+  visibility: hidden;
+  width: 15%;
+  background-color: var(--light-yellow);
+  opacity: 0;
+  border-radius: 0px 0px 15px 15px;
+  transition: background-color 0.2s, border-radius 0.2s, opacity 0.2s, visibility 0.2s;
+}
+
+.clearDropdownContent {
+  background-color: transparent !important;
+  border: 0.15vw solid var(--white);
+  border-top-right-radius: 15px;
+}
+
+.dropdownItem {
+  position: relative;
+  display: flex;
+  justify-content: left;
+  align-items: center;
+  padding: 5% 15% 0% 15%; /* no bottom padding */
+  transition: color 0.3s, opacity 0.2s, visibility 0.2s;
+}
+
+.dropdownItem:hover {
+  color: var(--red);
+  cursor: pointer;
+}
+
+.dropdownItem:hover a {
+  color: var(--red);
+  transition: color 0.3s, opacity 0.2s, visibility 0.2s;
+}
+
+.dropdownItemEnd {
+  position: relative;
+  display: flex;
+  justify-content: left;
+  align-items: center;
+  padding: 5% 15% 8% 15%;
+  transition: color 0.3s, opacity 0.2s, visibility 0.2s;
+}
+
+.dropdownItemEnd:hover {
+  color: var(--red);
+  cursor: pointer;
+}
+
+.dropdownItemEnd:hover a{
+  color: var(--red);
+  transition: color 0.3s, opacity 0.2s, visibility 0.2s;
+}
+
+.currentPageBold {
+  font-weight: bold;
+}
+
+.icon {
+  margin-left: -10px;
+  margin-right: 10px;
+  transform: scale(0.9);
+}
+
+/* compact navbar */
+.compactNavBarGrid {
+  display: grid;
+  grid-template:
+    'logo empty empty empty empty empty menuIcon';
+  align-items: center;
+  height: 100%;
+}
+
+.menuIconGridItem {
+  grid-area: menuIcon;
+}
+
+.menuButton {
+  position: relative;
+  display: flex;
+  justify-content: right;
+  align-items: center;
+  height: 100%;
+  width: 100%;
+  transition: color 0.3s;
+
+  background-color: inherit;
+  border: none;
+  color: inherit;
+  font: inherit;
+  text-transform: inherit;
+  padding: none;
+}
+
+.menuButton:hover {
+  color: var(--red);
+  cursor: pointer;
+}
+
+.menuIcon {
+  margin-left: -10px;
+  margin-right: 10px;
+  transform: scale(1.5);
+}
+
+.menuContainer {
+  display: block;
+  justify-content: center;
+  align-items: center;
+  margin: auto;
+  width: 100%;
+  font-size: 16px;
+  font-family: "Montserrat", sans-serif;
+  text-transform: uppercase;
+  transition: opacity 0.2s;
+}
+
+.hiddenMenu {
+  visibility: hidden;
+  opacity: 0;
+}
+
+.menuContent {
+  position: absolute;
+  width: 100%;
+  background-color: var(--light-yellow);
+  border-radius: 0px 0px 15px 15px;
+  transition: background-color 0.2s, border-radius 0.2s, visibility 0.2s;
+}
+
+.clearMenuContent {
+  background-color: rgba(255, 255, 255, 0.2) !important;
```

Comment:
Is there a reason why we're using `!important` in both CSS files?

Reply:
We used !important only to change the background colour of the dropdowns and text hover colour for dropdown container. Without using !important, we couldn’t find a way to make the background colour transparent or clear white colour (in compact navbar) when the scroll is at the top of the page.

Reply:
Thank you! Epic explanation 👍